### PR TITLE
adding pyTMD package

### PR DESCRIPTION
**Pull request by tsutterley**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/pyTMD/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (dopplershift) [inline recipes/pyTMD/LICENSE:1]: This is included in the PyPI tarball so you shouldn't include it here.

File: recipes/pyTMD/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
+{% set name = "pyTMD" %}
+{% set version = "1.0.3.0" %}
+{% set sha256 = "1ec3aa3afdb701a47f6096f199fdf7e9337624902e7d538de87c4a379ed72333" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyTMD-{{ version }}.tar.gz
```
Comment (dopplershift) [inline recipes/pyTMD/meta.yaml:10]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
This looks in pretty good shape, just a couple minor things. @mrsiegfried are you ok with being listed as a maintainer on this recipe?

File: recipes/pyTMD/LICENSE (hunk 1 above)
Comment (tsutterley) [reply recipes/pyTMD/LICENSE]: sounds good :+1:

**Review by dopplershift — APPROVED** [review]
Looks good. Just need to hear from @mrsiegfried that they are willing to be a maintainer for this recipe. Or you can remove them from the list of maintainers in the recipe and add them later in the feedstock once there is confirmation.

**Comment by mrsiegfried** [comment]
Sorry for not getting in here before @dopplershift. It’s been a long spring! I’m willing to be a maintainer.

**Comment by dopplershift** [comment]
No worries!

**Pull request merged**